include textmate grammars ext by default

Motivated by https://github.com/theia-ide/theia/issues/4979

I tried adding the dependency in package.json under brower-app or hello-world-extension, neither works, here is my package.json:

```json
"dependencies": {
    "@theia/core": "latest",
    "@theia/filesystem": "latest",
    "@theia/workspace": "latest",
    "@theia/preferences": "latest",
    "@theia/navigator": "latest",
    "@theia/process": "latest",
    "@theia/terminal": "latest",
    "@theia/editor": "latest",
    "@theia/languages": "latest",
    "@theia/markers": "latest",
    "@theia/monaco": "latest",
    "@theia/typescript": "latest",
    "@theia/messages": "latest",
    "@theia/textmate-grammars": "next",
    "hello-world-extension": "0.0.0"
  },
```

after I yarn, then yarn start, I got this error in browser:
```javascript
index.js:62 Error: Cannot apply @injectable decorator multiple times.
    at injectable.js:8
    at DecorateConstructor (Reflect.js:541)
    at Object.decorate (Reflect.js:130)
    at _decorate (decorator_utils.js:42)
    at Object.decorate (decorator_utils.js:55)
    at Object.../node_modules/@theia/textmate-grammars/node_modules/@theia/core/lib/browser/widgets/widget.js (widget.ts:25)
    at __webpack_require__ (bootstrap:63)
    at Object.../node_modules/@theia/textmate-grammars/node_modules/@theia/core/lib/browser/widgets/index.js (index.ts:17)
    at __webpack_require__ (bootstrap:63)
    at Object.../node_modules/@theia/textmate-grammars/node_modules/@theia/core/lib/browser/dialogs.js (dialogs.ts:20)
```

I'm using Chrome. You can try reproduce it just following this document:
https://www.theia-ide.org/doc/Authoring_Extensions.html

--- a/templates/app-package.json
+++ b/templates/app-package.json
@@ -14,6 +14,7 @@
     "@theia/languages": "<%= params.theiaVersion %>",
     "@theia/markers": "<%= params.theiaVersion %>",
     "@theia/monaco": "<%= params.theiaVersion %>",
+    "@theia/textmate-grammars": "<%= params.theiaVersion %>",
     "@theia/typescript": "<%= params.theiaVersion %>",
     "@theia/messages": "<%= params.theiaVersion %>",
     "<%= params.extensionName %>": "<%= params.version %>"
@@ -22,7 +23,7 @@
     "@theia/cli": "<%= params.theiaVersion %>"
   },
   "scripts": {
-    "prepare": "theia build --mode development",    
+    "prepare": "theia build --mode development",
     "start": "theia start",
     "watch": "theia build --watch --mode development"
   },

